apps/frontend/src/components/cln/BTCWallet/BTCWallet.test.tsx

```diff
@@ -1,11 +1,110 @@
 import { render, screen } from '@testing-library/react';
 import BTCWallet from './BTCWallet';
+import { AppContext } from '../../../store/AppContext';
+import { ApplicationModes, Units } from '../../../utilities/constants';
+
+const renderWithMockContext = (ui, { providerProps, ...renderOptions }) => {
```

Comment:
actually for `renderWithMockContext` i was going to move this somewhere and import it so its reusable, instead of writing every time in each test.  will do it later tonight.

Reply:
Done, added to test-utilities.jsx